host/greatfet/commands/greatfet_chipcon.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+#
+# This file is part of GreatFET
+
+from __future__ import print_function
+
+import errno
+import sys
+import ast
+
+import greatfet
+from greatfet import GreatFET
+from greatfet.utils import log_silent, log_verbose
+from greatfet.programmers.chipcon import ChipconProgrammer
+
+
+def main():
+    from greatfet.utils import GreatFETArgumentParser
+
+    # Set up a simple argument parser.
+    parser = GreatFETArgumentParser(description="Utility for chipcon debugging via GreatFET")
+    parser.add_argument('-id', '--chip_id', action='store_true',
+                        help="Print the chip ID of the connected device.")
+    parser.add_argument('-a', '--address', nargs=1, type=ast.literal_eval,
+                        help="Flash memory address to begin reading from / writing to")
+    parser.add_argument('-r', '--read', type=int,
+                        help="Number of bytes to read from flash memory")
+    parser.add_argument('-w', '--write_page', nargs='*', type=ast.literal_eval,
+                        help="Data to write to the specified flash page")
+    parser.add_argument('-p', '--program_flash',
+                        help="File name from which data will be written to flash from")
+    parser.add_argument('-e', '--erase', default=True,
+                        help="Specify whether or not the flash needs to be erased before writing")
+    parser.add_argument('-m', '--mass_erase', action='store_true', help="Erase the entire flash memory")
+    parser.add_argument('-rp', '--read_page', action='store_true', help="Read on page of flash memory")
+    parser.add_argument('-V', '--verify', nargs=1, default=True,
```

Comment:
`nargs=1` means that it expects an argument after the `--verify`. Trying this with `greatfet chipcon --verify` prints:
```
usage: greatfet_chipcon [-h] [-s <serialnumber>] [-i <i>] [--wait] [-v] [--chip-id] [-a ADDRESS] [-r READ]
                        [--write-page [WRITE_PAGE [WRITE_PAGE ...]]] [-w FLASH] [--no-erase] [-E] [--read-page]
                        [-V VERIFY]
greatfet_chipcon: error: argument -V/--verify: expected 1 argument
```

But also, since the default is True, what I said for `--erase` applies here too, so this should have `action='store_false'`, and the option should be changed to `--no-verify`.